app/content/pages/use-cases/drawing-on-images.html.erb

```diff
@@ -0,0 +1,12 @@
+---
+title: Drawing on Images
```

Comment:
Could we maybe include "HTML canvas" or just "canvas" in the title as well?

Reply:
for sure :)